compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/ExpectedTypeFinder.java

```diff
@@ -1097,5 +1130,10 @@ private Optional<TypeSymbol> getExpectedTypeFromFunction(BLangInvocation bLangIn
             return getParamType(bLangInvocation, argumentIndex, namedArgs);
         }
         return getExpectedType(bLangExpression);
+
+    }
+
+    private boolean isParenMissing(Token openParen, Token closedParen) {
```

Comment:
Should we rename this method to isParenthesisMissing? At first glance, I thought you made a typo and the method name was isParentMissing, but then I realized the actual logic.